### PR TITLE
Proposed fix for issue #641

### PR DESCRIPTION
**Pull request by brunomnsilva**
This is noticeable in smaller screens, e.g., smartphone (used chromium developer tools).
The mentioned file creates a new navbar in the DOM and it does not allow to access the sidemenu options, because it overlaps the toggle buttom previously set in the parent twigs.

Suggested fix is to add a new empty block to @admin/components/dashboard/navbar.html.twig to be extended by root/templates/components/dashboard/navbar.html.twig.

This twig only adds a "LOGIN WITH ROOT ACCOUNT" to the navbar. 

Current UI: 
prior-fix

After proposed fix:
after-fix

Proposed fix in `admin/components/dashboard/navbar.html.twig`:
```
{% block dashboard_navbar %}
    <div class="navbar-custom-menu">
        <ul class="nav navbar-nav">
          <!-- additional elements (e.g., menus, messages) before the user account drop down -->
          {% block dashboard_navbar_extra %} {% endblock %}
          <!-- User Account: style can be found in dropdown.less -->
          {% include "components/user-card.html.twig" %}
        </ul>
    </div>
{% endblock %}
```

Proposed fix in `root/templates/components/dashboard/navbar.html.twig`:
```
{% extends "@admin/components/dashboard/navbar.html.twig" %}

{% block dashboard_navbar_extra %}
  <li><a href="#">{{ translate("HEADER_MESSAGE_ROOT") | upper }}</a></li>
{% endblock %}

```

### CODE REVIEW

**Comment by lcharette** [comment]
Looks like your fork wasn't synced with the main `develop` branch before submitting the pull request. Can you try merging in the latest changes in `userfrosting/develop` to avoid merge conflicts?

See: https://help.github.com/articles/syncing-a-fork/

**Comment by brunomnsilva** [comment]
I think everything is okay now!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Change Log
 
+<<<<<<< HEAD
```
Comment (lcharette) [inline CHANGELOG.md]: This shoudn't be there... Same in `app/defines.php`

Comment (brunomnsilva) [inline CHANGELOG.md]: I know, but I don't know how to resolve it without reverting changes and starting over. The `develop` branch was fetched after the fix. I still had to use `git merge -s recursive -Xtheirs upstream/develop` to fix the local conflicts. I never touched those files locally.

Comment (lcharette) [inline CHANGELOG.md]: Just edit those file manually in your fork.

Comment (brunomnsilva) [inline CHANGELOG.md]: Getting a git 101 course over here! LOL  Think its now correct.

**Pull request merged**